README.md

```diff
@@ -114,6 +115,39 @@ public function rules()
 }
 ```
 
+### `Currency`
```

Comment:
For clarity, could you add a few examples of valid currencies here?

Reply:
Added for both this and the existing country rule